src/__test__/components/OfficeSectionComponent.test.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import expect from 'expect';
+
+import OfficeSections from '../../components/OfficeSections/OfficeSectionsComponent';
+
+import officeSectionsList from '../../_mock/officeSections';
+
+describe('Renders <OfficeSections /> correctly', () => {
+  let wrapper;
+
+  const props = {
+    loadOfficeSections: jest.fn(),
+    isLoading: false,
+    officeSectionsList: officeSectionsList.results
+  };
+
+  beforeEach(() => {
+    wrapper = shallow(<OfficeSections {...props} />);
+  });
+
+
+  it('renders Loading component if isLoading is true', () => {
+    wrapper.setProps({ isLoading: true });
+    expect(wrapper.find('LoaderComponent').length).toBe(1);
+  });
+
+  it('renders Table component when Loading is false', () => {
+    wrapper.setProps({ isLoading: false });
+    expect(wrapper.find('Table').length).toBe(1);
+  });
+
+  it('renders ItemsNotFoundComponent component if office Sections is empty', () => {
+    wrapper.setProps({ isLoading: false, officeSectionsList: [] });
+    expect(wrapper.find('ItemsNotFoundComponent').length).toBe(1);
+  });
+
+  it('calls handleRowChange when a user tries to change row limit', () => {
```

Comment:
This test description is misleading. From the description, I was expecting you to simulate a click or something and confirm that handleRowChange was called. The test is instead calling handleRowChange directly